aries-site/src/examples/templates/dashboards/components/ItemCountList.js

```diff
@@ -0,0 +1,74 @@
+import { useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  List,
+  Paragraph,
+  Text,
+  ThemeContext,
+} from 'grommet';
+import { DashboardCardHeader } from '.';
+
+export const ItemCountList = ({
+  defaultItemProps,
+  items,
+  menuItems,
+  title,
+}) => {
+  const theme = useContext(ThemeContext);
+  const { body, header } = theme.card;
+
+  return (
+    <Card>
+      <CardHeader
+        pad={{ horizontal: header.pad, top: header.pad, bottom: 'none' }}
+      >
+        <DashboardCardHeader title={title} menuItems={menuItems} />
+      </CardHeader>
+      <CardBody pad={{ horizontal: body.pad, vertical: 'small' }}>
+        <List data={items} defaultItemProps={defaultItemProps}>
+          {datum => (
+            <ItemCount
+              title={datum.title}
+              description={datum.description}
+              count={datum.count}
+            />
+          )}
+        </List>
+      </CardBody>
+    </Card>
+  );
+};
+
+const ItemCount = ({ title, description, count, ...rest }) => (
+  <Box direction="row" justify="between" {...rest}>
+    <Box>
+      <Text weight="bold">{title}</Text>
+      <Paragraph size="small" margin="none">
+        {description}
+      </Paragraph>
+    </Box>
+    <Text size="xlarge">{count}</Text>
```

Comment:
It gets a little tight here between the description/count, only right before the breakpoint change (so minimal impact). Wondering if weshould wrap the count in a box with `flex={false}`.

<img width="373" alt="Screen Shot 2022-02-07 at 11 28 00 PM" src="https://user-images.githubusercontent.com/12522275/152874862-81ac5f65-05ef-42e7-9fee-1614a28dd5da.png">

Reply:
Addressed by adding `gap` to Box row.